tests/unit/browser/test_adblock.py

```diff
@@ -0,0 +1,143 @@
+# vim: ft=python fileencoding=utf-8 sts=4 sw=4 et:
+#!/usr/bin/env python3
+
+# Copyright 2014-2015 Florian Bruhin (The Compiler) <mail@qutebrowser.org>
+#
+# This file is part of qutebrowser.
+#
+# qutebrowser is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# qutebrowser is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with qutebrowser.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Tests for qutebrowser.browser.adblock"""
+
+import os
+import zipfile
+
+import pytest
+
+from qutebrowser.browser import adblock
+from qutebrowser.config import config
+from qutebrowser.utils import objreg
+
+# TODO Should I use it ? And how ?
```

Comment:
If you wanted to use it, you wouldn't need the code here - since it's in `conftest.py`, you could simply use the [fixture](http://pytest.org/latest/fixture.html) by adding a `default_config` argument to a test.

That being said, I wouldn't use it, as I think it's better when tests have the minimal set of config options available they actually need - and there's the `config_stub` fixture exactly for that purpose.

You can use it like this:

```python
def test_foo(config_stub):
    config_stub.data = {'content': {'host-blocking-whitelist': ...}}
    ...
```